testimages/linux/install-java.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euxo pipefail
+
+echo "AGENT_IMAGE value is: $AGENT_IMAGE"
+if [ -z "$AGENT_IMAGE" ]; then
+  echo "AGENT_IMAGE variable is not set"
+  exit 1
+fi
+
+sudo apt-get update && sudo apt-get install openjdk-17-jre-headless -y && java -version
```

Comment:
May as well use Java 21 now, so we do not need to edit this later when Jenkins requires 21+.